modules/tracking/samples/lasot_benchmark.py

```diff
@@ -0,0 +1,213 @@
+"""
+The benchmark for trackers in the "tracking" module of the "opencv-contrib"
+repository. For evaluations used testing set of the LaSOT dataset:
+https://cis.temple.edu/lasot/
+For evaluations used metrics from LaSOT and TrackingNet papers.
+Link to LaSOT paper: https://arxiv.org/abs/1809.07845
+Link to TrackingNet paper: https://arxiv.org/abs/1803.10794
+"""
+
+import numpy as np
+import cv2 as cv
+import argparse
+import os
+
+from opencv.samples.dnn.dasiamrpn_tracker import DaSiamRPNTracker
```

Comment:
I'd like to suggest to use this construction:
```python
import sys

path = cv.samples.findFile('samples/dnn/dasiamrpn_tracker.py')
sys.path.append(os.path.dirname(path))
from dasiamrpn_tracker import DaSiamRPNTracker

```

Reply:
OK